.cm/platform_labels.cm

```diff
@@ -150,10 +150,13 @@ platforms:
   - ide:
     name: 'ide'
     subprojects:
-      - 'platforms/ide/'
-      - 'subprojects/ide/'
-      - 'subprojects/tooling-api/'
-      - 'subprojects/tooling-api-builders/'
+      - 'platforms/ide/base-ide-plugins'
```

Comment:
❗ The change to `summary_table.cm` is correct.  Make the change to this file match and it'll be good to go.

Reply:
Done